src/textField/index.tsx

```diff
@@ -226,6 +230,7 @@ export default class TextField extends PureComponent<TextFieldProps, TextFieldSt
               onClick={buttonOnClick}
               tabIndex="-1"
               title={buttonTitle}
+              aria-hidden={isEmpty(buttonTitle)}
```

Comment:
is buttonTitle a string? in that case shouldn't you do `{!!buttonTitle}` instead?

Reply:
`isEmpty` is ok and is more readable I think.

```
> isEmpty('')
true
> isEmpty('plop')
false
```